src/fix/gwcFix.cpp

```diff
@@ -0,0 +1,938 @@
+#include "gwcFix.h"
+#include "sbfInterface.h"
+#include "utils.h"
+#include "fields.h"
+
+#include <sstream>
+
+extern const string FixHeartbeat = "0";
```

Comment:
These don't need to be extern they are not used outside this compilation uint are they?